tests/h/traversal/user_test.py

```diff
@@ -10,71 +10,55 @@
 
 
 class TestUserContext:
-    def test_acl_assigns_read_to_AuthClient_with_user_authority(self, factories):
-        user = factories.User(username="fiona", authority="myauthority.com")
-        res = UserContext(user)
-        actual = res.__acl__()
-        expect = [(security.Allow, "client_authority:myauthority.com", "read")]
-        assert actual == expect
```

Comment:
It's the test below. We test it by checking it actually grants the permission in the situations we expect, rather than by testing the ACL return values.

We need one of these tests or the other, but not both.